src/transformers/models/vision_encoder_decoder/modeling_vision_encoder_decoder.py

```diff
@@ -334,14 +335,13 @@ def from_encoder_decoder_pretrained(
         if encoder is None:
             if encoder_pretrained_model_name_or_path is None:
                 raise ValueError(
-                    f"No `encoder_model` is passed to kwargs: {kwargs_encoder}. "
-                    f"In this case make sure that `encoder_pretrained_model_name_or_path` defined"
+                    "If `encoder_model` is not defined as an argument, a `encoder_pretrained_model_name_or_path` has "
+                    "to be defined."
```

Comment:
Changed it to be the same as the corresponding occurrence in other encoder decoder models.